### ISSUE TITLE
[1-17-6-7-54][Minor] Accessibility bar Settings stay the same after exiting a stage

### ISSUE THREAD

**Issue by Deckardsmegson** [whitelilydragon/ShangMuArchitect#56]
![grafik](https://user-images.githubusercontent.com/69774625/102001422-5f169600-3cf2-11eb-9ab2-9282657fca4e.png)
The screenshot shows what happens if you have the Move list enabled from Play mode and then switch to Editor mode. This also applies to all other features of the Accessibility Bar and since it cannot be accessed during Editor mode, it should probably be disabled.

**Comment by whitelilydragon** [comment]
That's a big oversight from before the last compiled build, whoops! It's fixed now.
